### PR TITLE
Add setup.py packaging, bugfixes and a bit of cleanup

### PR DESCRIPTION
**Pull request by benley**

### CODE REVIEW

File: systemd_wait.py [hunk 1]
```diff
@@ -52,46 +55,53 @@ def wait(bus, unit, target_states):
...
 
-    state = get_active_state()
-    if state not in target_states:
-        event_loop.run()
+    if get_active_state() not in target_states:
+        EVENT_LOOP.run()
 
     manager_int.Unsubscribe()
 
-    return state
```
Comment (Stebalien) [inline systemd_wait.py]: So, `nonlocal state` is why this used to work. Unfortunately, doing this *this* way:

1. Introduces an additional dbus method call.
2. May *not* return the state that caused us to stop waiting. That means we could return a state to the user that *isn't* in the set of states they told us to wait for.

I'd just leave this as it was (maybe commenting on what we're doing here). I'd take a less "spooky magic at a distance" way of doing this if you can think of one but I can't think of any simple solutions.

**Review by Stebalien — CHANGES_REQUESTED** [review]
Thanks! Looks like some reasonable changes.

File: systemd_wait.py (hunk 1 above)
Comment (benley) [reply systemd_wait.py]: Oh wow, I wasn't even aware that `nonlocal` was a thing - I initially tried this with python 2.7 and mistakenly assumed it was a typo!  I'll put it back.

Comment (benley) [reply systemd_wait.py]: Done, should be ready for another look.

**Review by Stebalien — APPROVED** [review]

**Comment by Stebalien** [comment]
Thanks!

**Pull request merged**